pyscf/grad/mspdft.py

```diff
@@ -553,19 +552,23 @@ def _debug_csfs (xci, label, normalize=False):
                     log.debug (' Root %d', iroot)
                     for s, v in zip (strs[iroot], vecs[iroot]):
                         log.debug ('  |%s>: %s', s, v)
-        else: 
-            def _debug_csfs (*args, **kwargs): pass
+        else:
+            def _debug_csfs (*args, **kwargs):
+                pass
         def _debug_cispace (xci, label):
             log.debug ('%s', label)
             xci_norm = [np.dot (c.ravel (), c.ravel ()) for c in xci]
             try:
                 xci_ss = self.base.fcisolver.states_spin_square (xci,
                     self.base.ncas, self.base.nelecas)[0]
             except AttributeError:
-                nelec = sum (_unpack_nelec (self.base.nelecas))
+                from pyscf.fci.direct_spin1 import _unpack_nelec
+                nelec = sum (_unpack_nelec(self.base.nelecas))
+                # There is no function spin_square anywhere... @Matthew R. Hermes
                 xci_ss = [spin_square (x, self.base.ncas,
```

Comment:
@MatthewRHermes spin_square is not a function anywhere in this document or anywhere else in the *PySCF-forge* repo. Not sure what you want to do here then.